### ISSUE TITLE
New Blockquotes Design

### ISSUE THREAD

**Issue by JonBrooksX** [jetlej/HE-6.0-Bugs#263]
I want the blockquotes in articles to be the same as the quote section on the front page. 

![screenshot 2016-02-25 23 36 27](https://cloud.githubusercontent.com/assets/13107838/13338355/c0a3b550-dc18-11e5-8a7e-377987d8600d.png)

Thoughts @lostboyevsky @MSchirp ?

**Comment by MSchirp** [comment]
Please make a screenshot of how they are, and one of how you want them to be. Which is which? @JonBrooksX

**Comment by JonBrooksX** [comment]
Existing block quote design: 

![screenshot 2016-02-28 01 29 26](https://cloud.githubusercontent.com/assets/13107838/13376697/caa3450e-ddba-11e5-8863-4270f1fa0b5c.png)

Design I would like it changed to (quote section on homepage): 

![screenshot 2016-02-25 23 36 27](https://cloud.githubusercontent.com/assets/13107838/13376699/dce53556-ddba-11e5-840e-a9b726a275f5.png)

**Comment by MSchirp** [comment]
Just as wide, less wide? Just to be specific, the only think you want is the actual " design to be changed to the purple bigger one, correct?

**Comment by lostboyevsky** [comment]
I'm assuming he means same width as the current format, but with the font, font size, and quotation icon from the front page design.

**Comment by MSchirp** [comment]
Thats clear, thanks!

**Comment by JonBrooksX** [comment]
I don't think I'm the authority on quote design and was hoping this would be a discussion, hence my tagging and asking "thoughts"?

What do you guys think?

**Comment by lostboyevsky** [comment]
I think it's difficult to know what will look good unless I see potential options in an actual context --- ie as they would appear in an article, with our normal font surrounding them. I'd like to see what I just described and think it would be better than what we have currently, but I can't say much more than that till I see it in context.

**Comment by JonBrooksX** [comment]
Ok. This isn't going anywhere.

**Comment by MSchirp** [comment]
Hey Jon, I thought your comment and closing of the issue was not constructive at all. I personally don't see why it wasn't going anywhere, and I agreed with your initial idea of changing this. We could simply ask Deepak to code up Bates' interpretation and then check if that is what we want.

**Comment by MSchirp** [comment]
@JonBrooksX

**Comment by JonBrooksX** [comment]
Fair enough! Felt like it wasn't going anywhere to me. Let's do that then.

**Comment by JonBrooksX** [comment]
@lostboyevsky @MSchirp

**Comment by MSchirp** [comment]
Looks great @JonBrooksX , will ask Deepak to code this up once were done with all the backbone/ 80/20 stuff! Thank you for reconsidering the issue.

**Comment by lostboyevsky** [comment]
I like it, @JonBrooksX, though I think it looks perhaps a touch too big. I would go about 95% of that size. I also think the font could be a bit sleeker. Really love Brain Pickings' blockquote font and would copy that same font if possible:
![screen shot 2016-03-16 at 3 45 04 pm](https://cloud.githubusercontent.com/assets/9164601/13826497/4fe22fa6-eb8e-11e5-9d0d-2acc54c85784.png)

**Comment by JonBrooksX** [comment]
Thanks @MSchirp. 

Yeah, @lostboyevsky, BP is gorgeous. Just checked the site and its articles are narrower than HE's, so we could get away with a slightly bigger font than BP without it looking stuffed. But I like the font a lot. It's called "fira-sans," available on Google Fonts. 

While we're at it I think we should change our font to "ff-tisa-web-pro" which is what BP uses.

Thoughts?

**Comment by JonBrooksX** [comment]
Okay, that blockquote font is actually available on Google Fonts: https://www.google.com/fonts/specimen/Fira+Sans

We should be able to add that to the site.

**Comment by lostboyevsky** [comment]
@JonBrooksX I can get onboard with changing our body font. I still think the body font could be a few shades darker too. I'm not a huge fan of EB Garamond. It's decent, but I think we can do better than that.

**Comment by JonBrooksX** [comment]
Okay. It's quite difficult to discuss what a nice font looks like theoretically. Can you find an example of a font you love online?

We can have any font available in the google font library. 

This is the font headspace use for example: https://www.headspace.com/blog/2016/03/30/work-less-and-do-a-better-job/

**Comment by lostboyevsky** [comment]
@JonBrooksX, i like the headspace font, but i don't think it quite fits HE's vibe. that bookerly font is amazing. would definitely be down to use a font that looks very similar to that if we can find one.
